setup.py

```diff
@@ -97,6 +97,8 @@ class ExtraDependencies:
     ]
     grid = [
         "gridx-prescient>=2.2.1",  # idaes.tests.prescient
+        "scikit-learn",
```

Comment:
Just to check, what are these dependencies required for and are they absolutely necessary?